taxcalc/functions.py

```diff
@@ -262,11 +306,11 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900, e19700,
 
     # Casualty
     if e20500 > 0:
-        c37703 = e20500 + ID_Casualty_frt * _posagi
-        c20500 = c37703 - ID_Casualty_frt * _posagi
+        c37703 = e20500 + ID_Casualty_frt * max(0, e00100)
```

Comment:
Note `_posagi = max(c00100, 0)`, by adopting an e-code here would definitely improve results. But I think we might still want to use calculated `c00100` here instead of `e00100`.

cc @MattHJensen